src/app/components/dashboard/TiplineDashboard.js

```diff
@@ -113,6 +113,7 @@ TiplineDashboard.propTypes = {
     get_language: PropTypes.string.isRequired,
     get_languages: PropTypes.string.isRequired,
     slug: PropTypes.string.isRequired,
+    smooch_bot: PropTypes.object.isRequired,
```

Comment:
[question] why is this line being added?

Reply:
From your comment in the ticket, I'm inferring this was introduced but was no longer necessary. If so, please remove this line.

Also, can you please change the `data_report` propType to `array` instead of `object`? Not related to this ticket but I noticed a propType validation error while testing and it would be great to fix it in this PR! Thanks!